manifests/cf-manifest/outputs/cf-secrets.yml

```diff
@@ -0,0 +1,207 @@
+---
```

Comment:
Do we really want to have cf-secrets.yml in github? Maybe add to .gitignore? Also, IMO, this file should never be created within the repository at all...